package.json

```diff
@@ -16,9 +16,10 @@
     "api:testWatch": "cd api && npm run testWatch",
     "api:watch": "cd api && tsc --watch",
     "api:wsl": "nodemon --legacy-watch api/bin/www",
-    "client": "cd client && npm run start",
```

Comment:
Should we put back the "client" script but make it run the "prod" instead of "start" script? I think this would be consistent with the current "backend" script, which runs the backend in production mode.

Reply:
okay.

Reply:
Made change.